src/case_runners/case_runner.jl

```diff
@@ -92,8 +93,11 @@ function run_genx_case_multistage!(case::AbstractString, mysetup::Dict)
     ### Cluster time series inputs if necessary and if specified by the user
     tdr_settings = get_settings_path(case, "time_domain_reduction_settings.yml") # Multi stage settings YAML file path
     TDRSettingsDict = YAML.load(open(tdr_settings))
-    TDRpath = joinpath(case, "Inputs", "Inputs_p1", mysetup["TimeDomainReductionFolder"])
+
+    first_stage_path = joinpath(case, "Inputs", "Inputs_p1")
+    TDRpath = joinpath(first_stage_path, mysetup["TimeDomainReductionFolder"])
     if mysetup["TimeDomainReduction"] == 1
+        prevent_doubled_timedomainreduction(first_stage_path)
```

Comment:
Here, we only check the configuration of the first stage of a multi-stage problem.
The other stages are checked in the `cluster_inputs` function.